test_regress/t/t_hier_block0_bad.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2003 by Wilson Snyder. This program is free software; you can
+# redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+
+scenarios(linter => 1);
+
+lint(
+    fails => 1,
+    verilator_flags2 => ['--stats', '--hierarchical'],
```

Comment:
```suggestion
    verilator_flags2 => ['--hierarchical'],
```

(Don't think --stats matters for this test). Perhaps similar change in other tests?

Reply:
Thanks
Unnecessary --stats is removed in fe13d48c76595dd8dfaa5ea95c1ed73de6c375e4